Gemfile.lock

```diff
@@ -214,6 +228,7 @@ DEPENDENCIES
   rspec_api_documentation
   sass-rails (~> 5.0)
   sqlite3
+  twitter-bootstrap-rails
```

Comment:
I have a custom bootstrap implementation that's being loaded in from assets, not major though. I can resolve leter.